myapp/templates/single_stock.html

```diff
@@ -61,64 +61,81 @@
-			<a class="nav-link" data-toggle="pill" href="#6m" role="tab" aria-controls="pills-contact" aria-selected="false">6M</a>
-		  </li>
-			<li class="nav-item">
-			<a class="nav-link" data-toggle="pill" href="#1y" role="tab" aria-controls="pills-contact" aria-selected="false">1Y</a>
-		  </li>
-			<li class="nav-item">
-			<a class="nav-link" data-toggle="pill" href="#max" role="tab" aria-controls="pills-contact" aria-selected="false">Max</a>
-		  </li>
-		</ul>
-		-->
+        <div id ="chartContainer">
+            <canvas id="stockChart"></canvas>
+        </div>
+        <ul class="nav nav-pills mb-3" id="pills-tab" role="tablist">
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('1d')" role="tab" aria-controls="pills-home" aria-selected="true">1D</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('5d')" role="tab" aria-controls="pills-home" aria-selected="true">5D</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link active" data-toggle="pill" onclick="getHistoricData('1m')" role="tab" aria-controls="pills-profile" aria-selected="false">1M</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('3m')" role="tab" aria-controls="pills-contact" aria-selected="false">3M</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('6m')" role="tab" aria-controls="pills-contact" aria-selected="false">6M</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('1y')" role="tab" aria-controls="pills-contact" aria-selected="false">1Y</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" data-toggle="pill" onclick="getHistoricData('max')" role="tab" aria-controls="pills-contact" aria-selected="false">Max</a>
+            </li>
+        </ul>
 	</div>
 </div>
 {% endblock %}
 
 {% block scripts %}
 <script>
-	$(document).ready(function(){
-		$.get( "/historic/{{ data.symbol }}/", function( data ) {
-			historic_data = data.data.sort(function(a, b) { return a.date - b.date; })
-
-			var ctxL = document.getElementById("stockChart").getContext('2d');
-			var myLineChart = new Chart(ctxL, {
-				type: 'line',
-				data: {
-					labels: historic_data.map(d => d.label),
-					datasets: [
-						{
-							label: "{{ data.symbol }}",
-							data: historic_data.map(d => d.close),
-							backgroundColor: [
-								'rgba(105, 0, 132, .2)',
-							],
-							borderColor: [
-								'rgba(200, 99, 132, .7)',
-							],
-							borderWidth: 2
-						}
-					]
-				},
-				options: {
-					responsive: true
-				}
-			});
-		});
-    });
+    $(document).ready(getHistoricData());
```

Comment:
small note, the JavaScript getting bigger here in this file, preferred to be separated and to include into a new JSscript file in `static/js/` folder